src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -136,24 +127,24 @@ export class PostItCollectionsComponent implements OnInit, OnDestroy {
     });
   }
 
-  public toggleFavorite(collection: Collection) {
-    this.collectionService.toggleCollectionFavorite(collection)
+  public toggleFavorite(collection: CollectionModel) {
+    this.collectionService.toggleCollectionFavorite(CollectionConverter.toDto(collection))
```

Comment:
We should not call services directly from components anymore. Try to update the collection object by dispatching an action.

Or is this some kind of workaround before favorites are implemented on the back-end? If so, this code should rather be moved to the store.

Reply:
This is more of a workaround. The task was to make the search-all perspective. The collection perspective changes are there only to react to query changes when searching.